ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -55,6 +55,16 @@ tests:
   openshift_installer:
     cluster_profile: gcp
     upgrade: true
+- as: e2e-aws-sdn-single-step-registry
```

Comment:
Step registry will be the only way to write these in the future, so you can ditch that suffix

Reply:
ok